.travis.yml

```diff
@@ -4,6 +4,10 @@ jdk:
   - oraclejdk7
   - oraclejdk8
 
+script:
+  - mvn clean -Pstrict compile test-compile -B
+  - mvn clean test -B
```

Comment:
let's invert the test phase and strict compile phase, so we only recompile the code twice instead of 3 times.
(and remove the clean step as part of the test to leverage the install done by travis before running script)